Videos

tamigoodall opened the pull request:
<!---
Notes:
- Your PR should conform to our [Contributing Guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md)
- If applicable, add details to the [update feed](https://github.com/auth0/docs/tree/master/updates)
- Make sure your PR gets deployed to a Heroku [Review App](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#review-apps) without errors
- Please include a short description
- If your PR is a work in progress, title it [DO NOT MERGE]
--->

tamigoodall committed:
Video landing pages

Started index.md page
Started first 2 video pages - 10 more to go

tamigoodall committed:
Update index.md

Updated new titles

tamigoodall committed:
Update 01_architecture-your-tenant.md

Fixed links

tamigoodall committed:
Update 02_provision-user-stores.md

Added provision user stores

tamigoodall committed:
Created subfolder

Created subfolder

tamigoodall committed:
Added index.md

Rename folder
Added index.md

tamigoodall committed:
New path added

new path added

tamigoodall committed:
Update 01_architecture-your-tenant.md

Collapsible transcript

tamigoodall committed:
Update 02_provision-user-stores.md

Collapsible transcript

tamigoodall committed:
Merge branch 'master' into videos

tamigoodall committed:
New index pages

New index pages

tamigoodall committed:
Added video image

Added video image

tamigoodall committed:
Added 03-provision-import-users

Changed png to jpg
Changed filenames
Updated links

tamigoodall committed:
Update sidebar.yml

Sidebar updates

tamigoodall committed:
Added first two video links

Added first two video links

tamigoodall committed:
Update 01-architecture-your-tenant.md

Fixed typo in video id

tamigoodall committed:
Create 04_01-authenticate-how-it-works.md

Added authenticate part 1

tamigoodall committed:
Added new content and updated sidebar and menu

Added more descriptions
Added Authenticate video pages
Updated sections.yml
Updated main index page

tamigoodall committed:
Moved Previous videos and other stuff

Added missing video to sidebar
Moved Previous videos to the bottom of pages
Added back in Previous videos descriptions
Added 05_01-authorize-id-tokens-access-control

tamigoodall committed:
Added video short descriptions and Authz part 2

Added video short descriptions and Authz part 2 page

tamigoodall committed:
Update 04_01-authenticate-how-it-works.md

Missing bullets

tamigoodall committed:
User Profiles

Added new user profiles doc
Fixed description for Branding
Added new video ids

tamigoodall committed:
Add Brand Parts 1 and 2

Add Brand Parts 1 and 2

tamigoodall committed:
Add Brand Part 3 and Logout

Added Brand Part 3 and Logout
Updated main index
Fixed incorrect description
Fixed Brand Part 2 links

tamigoodall committed:
Minor changes

Fixed filename errors
Cleaned up main index wording

tamigoodall committed:
Update index.md

Added topics

tamigoodall committed:
Update index.md

Fixed title

tamigoodall committed:
Video time changes

Video time changes

tamigoodall committed:
Updated transcript text

Fixed typos
Added missing text formatting

tamigoodall committed:
Changed video times and removed image

Changed video time
Misc text formatting
Fixed <summary> </summary> spacing
Removed main image
Added new intro text on main index page

tamigoodall committed:
Merge branch 'master' into videos

rkhoriander commented on articles/videos/get-started/05_02-authorize-get-validate-id-tokens.md:
```diff
@@ -0,0 +1,181 @@
+description: How to get and validate ID Tokens before storing and using them.
+classes: video-page
+---
+# Authorize: ID Tokens and Access Control
```
Change this to match the section heading in the left nav?

rkhoriander commented on articles/videos/get-started/08-brand-emails-error-pages.md:
```diff
@@ -0,0 +1,177 @@
+---
+# Brand: Emails and Error Pages
+
+Learn how to use Auth0 email templates and make changes to the reply email address, subject, redirect URL, and URL lifetime. You can configure your email provider and how verify emails look. Learn how to use your own error pages.
```
"how verify emails look" seems awkward. Maybe "customize how verify emails look"?

rkhoriander commented on articles/videos/get-started/07_01-brand-how-it-works.md:
```diff
@@ -0,0 +1,148 @@
+---
+# Brand: How It Works
+
+Learn how branding works with Auth0 and how to add your company name and logo as well as choose colors for your pages.
```
Awkward. Maybe "Learn how branding works with Auth0. Add your company name and logo, and choose colors for your pages."?

rkhoriander commented on articles/videos/get-started/05_01-authorize-id-tokens-access-control.md:
```diff
@@ -0,0 +1,139 @@
+---
+# Authorize: ID Tokens and Access Control
+
+Learn about using Auth0 authorization via the use of ID Tokens and understand what an ID Token is and how to add custom claims to customize them and make access control decisions for your users.
```
A lot packed into this sentence. Maybe "Learn about Auth0 authorization via the use of ID Tokens. Understand what an ID Token is, and learn how to add custom claims to customize tokens and make access control decisions for your users."

tamigoodall replied:
Much better

rkhoriander commented on articles/videos/get-started/04_01-authenticate-how-it-works.md:
```diff
@@ -0,0 +1,159 @@
+---
+# Authenticate: How It Works
+
+Learn about the difference between authentication, authorization, and access control. Understand when and why you might use each type of authentication method: First factors, second factors, and multi-factor. Learn about the OpenID Connect authentication protocol.   
```
Don't capitalize "First". For consistency with other pages,  add "(OIDC)" after "OpenID Connect".

rkhoriander commented on articles/videos/get-started/03-provision-import-users.md:
```diff
@@ -0,0 +1,198 @@
+---
+# Provision: Import Users
+
+Learn how Auth0 allows you to move your existing users to an Auth0 user store and why you may want to use a combination of both automatic and bulk migration methods. Learn why its a good idea to have an API between the cloud and your legacy database and how custom database scripts work.
```
it's

rkhoriander commented on articles/videos/get-started/02-provision-user-stores.md:
```diff
@@ -0,0 +1,190 @@
+---
+# Provision: User Stores
+
+Learn how user profiles are provisioned within an Auth0 tenant. Users can come from a multitude of places: directly through creating an account using self sign up, indirectly through using their favorite trusted social media platform, or collectively through a trusted enterprise directory. Auth0 makes it simple to quickly connect your tenant to multiple types of user stores. We group these users  ...
```
sign-up

rkhoriander commented on articles/videos/get-started/02-provision-user-stores.md:
```diff
@@ -0,0 +1,190 @@
+---
+# Provision: User Stores
+
+Learn how user profiles are provisioned within an Auth0 tenant. Users can come from a multitude of places: directly through creating an account using self sign up, indirectly through using their favorite trusted social media platform, or collectively through a trusted enterprise directory. Auth0 makes it simple to quickly connect your tenant to multiple types of user stores. We group these users  ...
```
We always seem to leave off legal identity as a user store. Maybe this doesn't matter, but feels like it's something missing, particularly for European users.

tamigoodall replied:
I'll add this to the list for the next revision.

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+---
+# Architecture: Your Tenant
+
+Learn what an Auth0 tenant is and how to configure it in the Auth0 Dashboard. Understand why you may want more than one tenant if you have different user communities, and also how you can use more than one tenant to support your Software Development Life Cycle. Learn the importance of tenant naming, use of custom domains, provisioning additional tenant administrators, and how to associate tenants ...
```
For consistency with other pages, add "(SDLC)" after "Software Development Life Cycle".

Last sentence is awkward. Missing verbs or an "and" somewhere?

tamigoodall replied:
Fixed

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+description: What an Auth0 tenant is and how to configure it in the Auth0 Dashboard.
+classes: video-page
+---
+# Architecture: Your Tenant
```
Small quibble, but (almost) all the other sections begin with a verb. This one begins with a nominalization. Aren't they actually "architecting" in this step when choosing how to use their tenants? So wouldn't "Architect" be the proper word?

tamigoodall replied:
Yes - much better.

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+  
+  In this video, we will talk about what an Auth0 tenant is, and show you how to configure it in the Auth0 Dashboard. We will also briefly explain why you may want to use more than one tenant if you have different user communities, and also how you can use more than one tenant in support of your Software Development Life Cycle. We’ll also describe the importance of tenant naming, use of custom do ...
+
+  Once you create your account in Auth0 you will be asked to create a Tenant. This is a logical isolation unit. The term tenant is borrowed from the phrase "software multi-tenancy" and refers to an architecture where a single instance of the software serves multiple tenants. No tenant can access the instance of another tenant, even though the software might be running on the same machine (hence t ...
```
"Once you create your account in Auth0,"

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+
+  When it comes to figuring out how to define your Auth0 tenants and accounts as part of your application integration, the value of investing time in the architectural “landscape” up front will pay dividends in the long run, and there are a number of things you’ll want to consider. 
+
+  It’s important to understand how your applications need to function within your infrastructure, and this will help you understand how to configure your tenants to accomplish your goals. How your Auth0 tenants are configured -- the architecture of your Auth0 deployment -- will form the basis for the grouping of your Auth0 assets to leverage features such as Single Sign On, centralized user profi ...
```
I believe we use the `&mdash;` HTML code for em dashes.

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+
+  So you will need to decide how many different production tenants you will require. Most companies only need a single production tenant, but if you have a set of applications for just employees, you may want a separate tenant for that. In some situations, you may also need different production tenants for different customer user communities. 
+
+  Another reason you would want to create multiple tenants is to support your Software Development Life Cycle. Auth0 can fit into your process by allowing you to have a separate tenant for each phase - such as one for development, one for testing, and one for production. Even if you don’t use an SDLC methodology, you will most likely want to create at least two tenants: one for development and on ...
```
Should be an em dash here.

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+
+  Another reason you would want to create multiple tenants is to support your Software Development Life Cycle. Auth0 can fit into your process by allowing you to have a separate tenant for each phase - such as one for development, one for testing, and one for production. Even if you don’t use an SDLC methodology, you will most likely want to create at least two tenants: one for development and on ...
+
+  You may want to name one tenant “company-dev” to serve as a shared environment where your development work occurs, and name another tenant “company-qa” for testing your Auth0 integration. You can then name a third tenant “company-prod”, to serve as your production tenant. 
```
No comma necessary after "company-prod".

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+
+  You can also create tenants to serve as sandboxes to test potential changes, like different deployment scripts, without compromising your environment. Auth0 lets you create as many free tenants as you like, but you may be limited for the number of tenants where all paid features are enabled. You can have up to three tenants where all features are available.
+
+  It’s also important to consider exactly what you will name your tenants. The ultimate goal is to end up with one or more production tenants that are branded exactly the way you want them to appear to your users. Tenant naming patterns are very important, so it’s good to plan the tenant names in advance because you won’t be able to change them once you create them,or use them again if you delete ...
```
Missing space before "or".

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+
+   Starting with the General settings tab, you can specify your company name, a path to your company logo, and your company’s support email address and support URL. This is the information that is shown in the default error page that appears to your users, so they can contact your support if they have an issue. You may also want to consider creating a custom error page and configure Auth0 to use  ...
+
+  Out-of-the-box, client facing URLs are Auth0 branded, however, we recommend using the Auth0 custom domain capability to provide a consistent corporate identity and to also address potential user confidence concerns before they arise. Your company name as part of the URL, for example, will support your brand and should tell your users that they can trust that they are in the right place to enter ...
```
"Out-of-the-box, client-facing URLs are Auth0 branded; however,"

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+
+  Out-of-the-box, client facing URLs are Auth0 branded, however, we recommend using the Auth0 custom domain capability to provide a consistent corporate identity and to also address potential user confidence concerns before they arise. Your company name as part of the URL, for example, will support your brand and should tell your users that they can trust that they are in the right place to enter ...
+
+  Having more than one Auth0 Dashboard administrator is a good idea, and periodically reviewing the list of Auth0 Dashboard administrators to see that the right people have access to your Auth0 tenants will help you make sure that each person has a legitimate need for admin access. You should make sure that former employees no longer have access. We also recommend that you enable multi-factor aut ...
```
Capitalize "If" in last sentence.

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+ 
+  There are also some advanced tenant settings that you can configure for your tenant. There are a couple of things you need to consider when you configure these items. 
+
+  Logout is the act of terminating an authenticated session. It is a security best practice to terminate sessions when they’re no longer needed to avoid a potential takeover by unauthorized parties. This is usually achieved via the provision of some “logout” option on the UI. There are multiple types of sessions that could be created when a user logs in. For example, a local application session,  ...
```
"third party" should be "third-party".

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+
+  Session timeout settings allow you to specify when the SSO cookie times out. The value you set is the login session lifetime which is how long the session will stay valid, measured in minutes. The default setting is set for 10,080 minutes or 7 days. 
+
+  Your contractual agreement with Auth0 should cover all the tenants you want to use. Make sure all your tenants are associated with your company account. If you have developers that want to create their own sandboxes for testing, make sure the tenants are associated with your account so they have the same permissions and Auth0 features available too. 
```
"developers that" should be "developers who" (although I guess this is a transcript, so if the person said "that", then I guess it stays?)

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+<details>
+  <summary>Summary</summary>
+  
+  So, to summarize, we described what an Auth0 tenant is, how to configure it, and why you may need more than one tenant. We  also described the importance of tenant naming, and how to associate tenants with your Auth0 accounts to save money and leverage Auth0 features across your organization.
```
Extra space after "We" in second sentence.

rkhoriander commented on articles/videos/get-started/01-architecture-your-tenant.md:
```diff
@@ -0,0 +1,159 @@
+  
+  So, to summarize, we described what an Auth0 tenant is, how to configure it, and why you may need more than one tenant. We  also described the importance of tenant naming, and how to associate tenants with your Auth0 accounts to save money and leverage Auth0 features across your organization.
+
+  In the next video, we’ll talk about provisioning your users and configuring user stores, and in a future video we’ll talk more about Custom Domains too.
```
Add comma after "in a future video".

tamigoodall committed:
Incorporated review comments

Incorporated review comments

tamigoodall committed:
Fixed em dashes and title error

Fixed em dashes with &emdash;
Fixed title error for Authorize: Get and Validate ID Tokens

tamigoodall committed:
Update 01-architecture-your-tenant.md

Fixed missing hyphen
Fixed back URL formatting
